src/test/java/core/basesyntax/FileWriterImplTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.service.FileWriter;
+import core.basesyntax.service.impl.FileWriterImpl;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FileWriterImplTest {
+    private static final String FIRST_REPORT =
+            "apple,100" + System.lineSeparator()
+            + "banana,200" + System.lineSeparator()
+            + "mango,300" + System.lineSeparator()
+            + "kiwi,400" + System.lineSeparator();
+    private static final String FIRST_REPORT_PATH =
+            "src/test/resources/output/report1.csv";
+    private static final String EMPTY_REPORT_PATH =
+            "src/test/resources/output/empty.csv";
+    private FileWriter fileWriter;
+    private File file;
+
+    @Before
+    public void setup() {
+        fileWriter = new FileWriterImpl();
+    }
+
+    @After
+    public void deleteFile() {
+        file.delete();
+    }
+
+    @Test
+    public void writeReport_ToFile_ok() {
```

Comment:
```suggestion
    public void writeReport_ExistInput_ok() {
```
the second parameter is a statement, correct the names everywhere